services/core-api/app/api/mines/reports/models/mine_report.py

```diff
@@ -126,10 +141,40 @@ def find_by_mine_report_guid(cls, _id):
     def find_by_mine_guid_and_category(cls, _id, category):
         try:
             uuid.UUID(_id, version=4)
-            reports = cls.query.filter_by(mine_guid=_id).all()
+            reports = cls.query.filter(
+                MineReport.permit_condition_category_code.is_(None)).filter_by(mine_guid=_id).all()
```

Comment:
should have a check for deleted_ind

Reply:
delete works as a hard delete for reports